src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -105,8 +114,8 @@
            :shows-horizontal-scroll-indicator false
            :nestedScrollEnabled               true
            :separator                         [rn/view {:width 12}]
-           :data                              communities
-           :render-fn                         community-list-item
+           :data                              (if loaded? communities [1 2 3])
```

Comment:
This `[1 2 3]` confused me a lot. It seems the values of the vector are ignored and the intention is to render 3 loading instances, right? If that's the case, we could give a name to the max number of elements and generate a sequence based on that.

```clojure
(if loaded? communities (range max-loading-instances))
```